app/models/edit_course_options.rb

```diff
@@ -0,0 +1,27 @@
+class EditCourseOptions
+  def initialize(course)
+    @course = course
+  end
+
+  def entry_requirements
+    Course::ENTRY_REQUIREMENT_OPTIONS
+      .reject { |a| a.include? %i[not_set not_required] }
+      .map do |choice|
```

Comment:
Unless I'm missing something, `Course::ENTRY_REQUIREMENT_OPTIONS` is a `key => value` pair, so should this be:

```suggestion
      .map do |choice, ucas_value|
```